plugins/lsp/language-servers/default.nix

```diff
@@ -416,6 +416,11 @@ with lib; let
       name = "metals";
       description = "metals for Scala";
     }
+    {
+      name = "nginx-language-server";
```

Comment:
`nginx-ls` (or simply `nginx`) would feel more consistent with (most) other LSP attr names, although there's already a handful that use the "full" `***-language-server` style...

Maybe we should consider providing less verbose and/or more consistent aliases? Not in scope here, of course.

Reply:
Actually, for the name, we use the ones that [nvim-lspconfigs](https://github.com/neovim/nvim-lspconfig/blob/master/doc/server_configurations.txt) gives. We only substitute `_` for `-` (`foo_bar` -> `foo-bar`).